.github/workflows/update-runners.yaml

```diff
@@ -0,0 +1,107 @@
+# This workflows polls releases from actions/runner and in case of a new one it
+# updates files containing runner version and opens a pull request.
+name: Update runners
+
+on:
+  schedule:
+    # run hourly
+    - cron: "0 * * * *"
```

Comment:
I'm thinking out loud here - running this hourly will be quite noisy here: https://github.com/actions/actions-runner-controller/actions 

Maybe we should rethink this frequency

Reply:
Do we want to try daily first? I'm down with that, I had started with hourly only to make this solution more compelling against the real-time solution :)